docs/readme.md

```diff
@@ -0,0 +1,23 @@
+# Building docs
+
+Most docs can be *prepared* locally, but the build process is partially invested in the github workflows.
+
+## First steps
+
+Prepare docker container for local builds
+
+```bash
+make_env.bash $TAG
```

Comment:
question: What is `$TAG`?

Reply:
Literally the tag for the image that Thrust/CUB shares for building, but yeah, I'll do that.